app/src/repositories/webtoon/episodeRepository.js

```diff
@@ -16,13 +17,25 @@ class EpisodeRepository {
         FROM episodes
         WHERE webtoon_id = ?;
         `;
-    try {
-      const [rows] = await pool.query(query, [webtoonId]);
-      return rows;
-    } catch (error) {
-      console.error("DB Error [getEpisodesByWebtoonId]:", error);
-      throw new Error("웹툰의 회차정보를 가져오는 데 실패했습니다.");
-    }
+    const [rows] = await pool.query(query, [webtoonId]);
+    return rows;
+  }
+
+  // 회차 상세정보 불러오기
+  async getEpisodeDetailById(episodeId) {
+    const query = `
+        SELECT
+          e.id AS episodeId,
+          e.episode_no,
+          e.title AS episodeTitle,
+          e.full_img_url,
+          w.title AS webtoonTitle
```

Comment:
return 시에 camelCase로 컨버팅해주기 때문에 RDB에서 값을 뽑아낼때는 snake_case로 통일시키는건 어떻게 생각하시나요??
row를 뽑아낸 뒤 바로 camelCase로 변환해주기는 하지만 중간에 어떤 로직이 추가될 때 헷갈릴 여지가 있을 수 있을거같다는 생각이 있습니다

```suggestion
          e.id AS episode_id,
          e.episode_no,
          e.title AS episode_title,
          e.full_img_url,
          w.title AS webtoon_title
```

Reply:
알아보기 쉽게 하려고 별칭을 주고 camelCase로 맞췄었는데, 쿼리문 안에서는 로직이 추가되거나 가공될 때 혼동을 방지하기 위해snake_case로 통일하는 게 더 명확할 것 같습니다.
피드백 감사합니다!